resources/Shaders/ModelExample/shader.vert

```diff
@@ -0,0 +1,26 @@
+#version 450
+
+layout (location = 0) in vec3 position;
+layout (location = 1) in vec3 color;
+layout (location = 2) in vec3 normal;
+layout (location = 3) in vec2 tex_coord;
+
+layout (location = 0) out vec3 fragColor;
+
+layout (set = 0, binding = 0) uniform Ubo {
+    //mat4 camera_projection;
+    mat4 model;
+    mat4 view;
+    mat4 proj;
+} ubo;
+
+//layout(push_constant) uniform Push {
+//    mat4 transform;
+//} push;
```

Comment:
What was this push constant used for? Is this an OpenGL vs Vulkan thing?

Reply:
I forgot to remove this lines. It's vulkan specific uniform we haven't implemented yet